packages/camera/camera_android/example/integration_test/camera_test.dart

```diff
@@ -205,6 +206,71 @@ void main() {
     expect(duration, lessThan(recordingTime - timePaused));
   });
 
+  testWidgets('Set description while recording', (WidgetTester tester) async {
+    final List<CameraDescription> cameras =
+        await CameraPlatform.instance.availableCameras();
+    if (cameras.length < 2) {
+      return;
+    }
+
+    final CameraController controller = CameraController(
+      cameras[0],
+      ResolutionPreset.low,
+      enableAudio: false,
+    );
+
+    await controller.initialize();
+    await controller.prepareForVideoRecording();
+
+    await controller.startVideoRecording();
+    sleep(const Duration(milliseconds: 500));
+
+    // SDK < 26 will throw a platform error when trying to switch and keep the same camera
+    // we accept either outcome here, while the native integration tests check the outcome based on the current Android SDK
```

Comment:
Nit: the native tests referred to here are native unit tests, not native integration tests.